Stripe/Payments/STPPaymentHandler.m

```diff
@@ -122,7 +127,11 @@ - (void)handleNextActionForPayment:(NSString *)paymentIntentClientSecret
         NSAssert(NO, @"Should not handle multiple payments at once.");
         completion(STPPaymentHandlerActionStatusFailed, nil, [self _errorForCode:STPPaymentHandlerNoConcurrentActionsErrorCode userInfo:nil]);
         return;
+    } else if (![STPPaymentIntentParams isClientSecretValid:paymentIntentClientSecret]) {
```

Comment:
@davidme-stripe overnight I was thinking of changing this to just be an assert in case for some reason the format changes in the future...what do you think? There are other places where we are depending on splitting the key

Reply:
I'd rather not add asserts that could trigger in other folks' apps in production if we change the format. Best to give them an error and let them handle it as they'd like.

Reply:
I mean debug only asserts so shipped sdks don't fail if there's a client secret format change 🙃 probably overly cautious

Reply:
I like the idea of debug-only asserts. Interesting idea to consider.